featurebyte/query_graph/node/generic.py

```diff
@@ -1827,6 +1833,53 @@ def _derive_sdk_code(
         return statements, ExpressionStr(f"{grouped}.{agg}")
```

Comment:
Maybe we could consider to ignore `backward` parameter here if it is default (since we are going to deprecate this).